autoload/neomake/makers/ft/go.vim

```diff
@@ -1,7 +1,15 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#go#EnabledMakers()
-    return ['go', 'golint']
```

Comment:
`gometalinter` sounds like it should be preferred over other linters (from the name of it) - similar to `pylama` on Python.